### DISCUSSION TITLE
personal-development-ordibehesht-1403/

### DISCUSSION THREAD

**Discussion by giscus** [amir-pourmand/hugo-papermod-farsi-template#41] [Announcements]
# personal-development-ordibehesht-1403/

ماه اردیبهشت هم به پایان رسید و وقت اینه که گزارش توسعه مهارتی این ماه رو ارائه بدم.
فعلاً اوضاع مرتبه و به طرز عجیبی نسبت به تهران از همه ابعاد بهتر شده. هم بیشتر و مفیدتر برای کار وقت می‌گذارم. هم خانواده‌ام رو بیشتر می‌بینم. هم مطالعه بیشتر می‌کنم و هم تفریح بیشتری دارم! قاعدتاً باید این وقت از جایی کم‌شده باشه و فکر می‌کنم اون بخش ترافیک و حاشیه‌های زندگی در تهرانه.

https://aprd.ir/personal-development-ordibehesht-1403/

**Comment by elhamghadiri** [comment]
سلام 
چند ماهی هست که وبلاگتون رو میخونم و یاد میگیرم. امروز هم به نکات جالبی برخوردم یکی روش agile تو یادگیری و دیگری هم انواع محتوای این روزهای دنیای دیجیتال.

**Reply by pourmand1376** [reply]
سلام الهام جان. 

خوشحالم که برات مفید بوده. 

کلا تفکر اجایل از مواردی هست که تو زندگیم رخنه کرده و هر جا می‌خوام کاری رو انجام بدم، سعی می‌کنم اجایل باشه؛ یعنی به جای این که یک سنگ بزرگ رو در نظر بگیرم، سعی می‌کنم اقدامات کوچکی تعریف کنم و انجام بدم و ببینم که چی پیش میاد. 

به تجربه دریافتم که اینطوری خروجی‌های منطقی‌تری بدست میاد. به نظر می‌تونه راهکار خوبی برای کمال‌گرایی هم باشه. 

یک پادکستی هم بود که راجع به اجایل صحبت کرده بود و خیلی دوستش داشتم. یادمه که تو وبلاگم هم یه جا بهش لینک داده بودم ولی پیداش نکردم :) پیداش کنم لینکش رو می‌گذارم.
